modules/aws-databricks-unity-catalog/uc_cross_account_role.tf

```diff
@@ -28,6 +28,19 @@ data "aws_iam_policy_document" "passrole_for_uc" {
       values   = ["arn:aws:iam::${var.aws_account_id}:role/${var.prefix}-uc-access"]
     }
   }
+  statement {
```

Comment:
This looks like it's already included in the statement above? `[sid = "ExplicitSelfRoleAssumption"` on lines 17 through 30](https://github.com/databricks/terraform-databricks-examples/pull/118/files#diff-61bf3103502ffe6d856ec15434c9d8b532d1ba9f1c4e3762f18f411d066e6352R17-R30)

Reply:
True. Removing.